### PR TITLE
Use CDN for JS and update meta data

### PR DESCRIPTION
**Pull request by AshleyPoole**
Use CDN for all public JS and making start on meta data

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,13 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <meta name="description" content="Meetup is a free responsive single page bootstrap template by designerdada.com">
-  <meta name="author" content="Akash Bhadange">
-  <title>Meetup: Free Responsive One Page Bootstrap Template</title>
+  <meta name="description" content="Web developer user group based in Shropshire, UK.">
+  <meta name="keywords" content="Developer,User Group,Developers,Programming,HTML,CSS,XML,JavaScript,Angular,DotNet,CSharp,C#,ASP,Telford,Wolverhampton,Shrewsbury">
```
Comment (AshleyPoole) [inline src/index.html]: @magicweasel we should chat about what keywords we want to use as well as the description... This is simply just a start...

File: src/index.html [hunk 2]
```diff
@@ -440,11 +444,11 @@ <h3 class="content-ct"><span class="ti-email"></span> Contact Form</h3>
       <!-- End: Footer -->
 
       <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
-      <script src="js/jquery.min.js"></script>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```
Comment (AshleyPoole) [inline src/index.html]: Moved all applicable to JS to use CDN called cdnjs.

File: src/index.html (hunk 1 above)
Comment (magicweasel) [inline src/index.html]: Yep cool, good start :)

File: src/index.html (hunk 2 above)
Comment (magicweasel) [inline src/index.html]: :)

**Pull request merged**